### ISSUE TITLE
'Choose another subject' is inaccessible with a screenreader, or with keyboard navigation

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#2300]
## Package
lib-classifier

## Describe the bug
The announcement that a subject is finished, and the options to choose another subject, are hidden away at the bottom of the page (after the footer.)

Here you can see VoiceOver reading out the task instruction when I enter the task area, although a sighted user gets a message that the subject is retired and not available.
<img alt="The task area in VoiceOver. The visible text says that the subject is finished, but VoiceOver is reading out the task instruction." src="https://user-images.githubusercontent.com/59547/125204130-8e36bb00-e273-11eb-8cfb-45c07f6c3727.jpg" width=200 />

## To Reproduce
1. Go to https://frontend.preview.zooniverse.org/projects/bogden/scarlets-and-blues/classify/workflow/17076/subject-set/96235/subject/63413209?finished=true
2. Read through the page in VoiceOver (or the screenreader of your choice.)
3. Alternatively, tab through the page to the task area.

## Expected behavior
When I enter the task area In a screenreader, I should hear "This subject has enough classifications, so any additional classifications submitted will not be used in its analysis. You can still view this subject for reference." and then options to choose a new subject.

When I tab through the page, I should tab into that popup when I reach the task area.

## Additional context
Adding content to the DOM at the end of the document is a common method for implementing modals. It kind of breaks for popups that aren't modal, because the rest of the page is still accessible via the keyboard. Ideally, the content of a targeted layer like this one would be inserted into the DOM at the start of the target element.

This same issue affects the content of the field guide, which should be accessible while also continuing to classify.

**Comment by eatyourgreens** [comment]
A skip link, saying “This subject is finished” and linking to the Finished Subject box, might work at the beginning of the task area.

**Comment by eatyourgreens** [comment]
Now that `<dialog>` is well-supported, this could be fixed by inserting the dialog into the task area, rather than the end of the document.
